tests/integration/examples/test_notebooks_python.py

```diff
@@ -246,3 +247,20 @@ def test_geoimc_integration(notebooks, output_notebook, kernel_name, expected_va
 
     for key, value in expected_values.items():
         assert results[key] == pytest.approx(value, rel=TOL, abs=ABS_TOL)
+
+
+@pytest.mark.integration
+@pytest.mark.skip(reason="Tests removed due to installation incompatibilities")
```

Comment:
We could add a separate pipeline for this test in the future, since xlearn can be pip installed on a DSVM.

Reply:
yeah, good suggestion, maybe consider it for a new PR?

Reply:
Yes, I created an issue about this https://github.com/microsoft/recommenders/issues/1437